gitgud/__main__.py

```diff
@@ -253,20 +258,31 @@ def handle_levels(self, args):
 
     def handle_load(self, args):
         self.assert_initialized(skip_level_check=True)
-        if args.skill_name in all_skills:
-            skill = all_skills[args.skill_name]
+        
+        argset = args.skill_name.split("-")
 
+        if (len(argset) == 2):
+            args.skill_name, args.level_name = tuple(argset)
+        elif(len(argset) > 2):
+            print("You put too many dashes! Use at most one dash when trying to load a level.")
+            return
+        
+        args.skill_name = int(args.skill_name) - 1 if args.skill_name.isnumeric() else args.skill_name
+        args.level_name = int(args.level_name) - 1 if (args.level_name is not None and args.level_name.isnumeric()) else args.level_name
```

Comment:
the `X if cond else Y` syntax is supposed to make things more easy to understand, do you think could change this? See comment in general review for more suggestions